app/controllers/empire/food.js

```diff
@@ -10,12 +10,16 @@ export default Controller.extend({
   isGenFoodAvailable: computed('model.type', function() {
     return this.model.type == "religious"
   }),
-  workerHunterAvailable: computed('model.type', 'game.upgrades.@each.isActive', function() {
-    return this.game.getUpgrade('Hunting').isActive && (this.model.type == "economical" || this.game.getUpgrade('Universal Worker').isActive)
-  }),
 
   foodStorageBuildings: filter('model.foodStorageBuildings', b => ! b.isCapital),
+  // This is ugly: Use sum to do a reduced `or`, because ember's functional sucks balls.
```

Comment:
Better: Do as in templates, and upgrade the filtering so that you can directly do `{{#if list}}`